src/main/java/com/easyrest/facade/ActionOnRestaurantFacade.java

```diff
@@ -0,0 +1,67 @@
+package com.easyrest.facade;
+
+import com.easyrest.components.ActionOnRestaurants;
+import com.easyrest.components.HeaderForAuthorizedUser;
+import com.easyrest.components.UserRoleMenuPanel;
+import com.easyrest.components.moderator.LeftSideMenu;
+import com.easyrest.pages.moderatorPanel.Restaurants;
+import org.openqa.selenium.WebDriver;
+
+public class ActionOnRestaurantFacade {
+    private WebDriver driver;
+    private ActionOnRestaurants actionOnRestaurants;
+    private LeftSideMenu leftSideMenu;
+
+    public ActionOnRestaurantFacade(WebDriver driver) {
+        actionOnRestaurants = new ActionOnRestaurants(driver);
+        leftSideMenu = new LeftSideMenu(driver);
+        this.driver = driver;
+    }
+
+    public void seeAllRestaurants() {
+        leftSideMenu.clickOnMenuButtonRestaurants();
+    }
+
+    public void clickUnapprovedRestaurant() {
+        leftSideMenu.clickOnMenuButtonRestaurants();
+        actionOnRestaurants.clickOnButtonUnapproved();
+    }
+
+    public void clickApprovedRestaurant() {
+        leftSideMenu.clickOnMenuButtonRestaurants();
+        actionOnRestaurants.clickOnButtonApproved();
+    }
+
+    public void clickArchivedRestaurant() {
+        leftSideMenu.clickOnMenuButtonRestaurants();
+        actionOnRestaurants.clickOnButtonArchived();
+    }
+
+    public boolean approveRestaurant() {
```

Comment:
you don't need these methods because you have a facade for that, please consider your changes to follow Page Objects and Facade patterns